Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Looks like inconsistent header levels.</p><p>*probabilities</p><p>Why bounded percentages? Just percentages.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Fixed header, all should be # top level</p><p> </p><p>Bounded percentage because beta can't model percentages over 100%, like 150% growth or something like that</p><p> </p>

Reply:
<blockquote>Fixed header, all should be # top level</blockquote><p>The convention is to have # be only the title. Then section headers are ## (this would be discrete and continuous) and ### for sub-section headers (this would be for the distributions).</p><p> </p><p>This actually matters e.g. when creating a TOC.</p>

Reply:
<p>For TOC yes but im optimizing for slides here with consistent header size</p><p> </p>